examples/python/HelloWorld/README.md

```diff
@@ -260,12 +263,39 @@ def _no_cache(request, context):
             yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
 ```
 
+### `EchoTable_3` function
+
+We just echo back exactly the information that was sent, and without adding table headers.
+Since we do not add table headers, the returned table gets the default column names Field1, Field2, Field3.
+
+``` python
+    @staticmethod
+    def _echo_table(request, context):
+        """
+        Echo the input table.
+        :param request:
+        :param context:
+        :return:
+        """
+        for request_rows in request:
+            response_rows = []
+            for row in request_rows.rows:
+                response_rows.append(row)
+            yield SSE.BundledRows(rows=response_rows)
+```
+
 ## Qlik documents
-An example document is given for Qlik Sense (SSE_Hello_World.qvf) and QlikView (SSE_Hello_World.qvw). The example consists of three sheets, one with script function calls and two with user defined function calls. The function calls on each of the first two sheets demonstrate the same functionality. We use a table for the tensor call and a KPI object for the aggregation call. On the third sheet we demonstrate enabling and disabling the cache for a specific function. A field called __HelloWorldData__ consisting of two rows of strings is loaded into the Qlik engine.
+An example document is given for Qlik Sense (SSE_Hello_World.qvf) and QlikView (SSE_Hello_World.qvw). The example consists of four sheets, one with script function calls, two with user defined function calls and one with the result of table load. The function calls on each of the first two sheets demonstrate the same functionality. We use a table for the tensor call and a KPI object for the aggregation call. On the third sheet we demonstrate enabling and disabling the cache for a specific function. A field called __HelloWorldData__ consisting of two rows of strings is loaded into the Qlik engine.
```

Comment:
We provide example documents for Qlik Sense (SSE_Hello_World.qvf) and QlikView (SSE_Hello_World.qvw).